partcad/src/partcad/assembly_factory_assy.py

```
@@ -31,6 +31,114 @@ def __init__(self, ctx, source_project, target_project, config):
+        # Read the body of the configuration file
+        fp = open(self.path, "r")
+        config = fp.read()
+        fp.close()
+
+        # Resolve Jinja templates
+        # NOTE: autoescape must stay off. The rendered document is YAML,
+        # not HTML, so escaping corrupts every parameter value that
+        # contains '&', '<', '>', '"' or "'" (e.g. 'a & b' would reach
+        # the parser as 'a &amp; b'). This matches how 'partcad.yaml'
+        # itself is rendered in 'ProjectLocal'.
+        template = Environment(
```

Ruff (0.16.1)

[error] By default, jinja2 sets `autoescape` to `False`. Consider using `autoescape=True` or the `select_autoescape` function to mitigate XSS vulnerabilities.

(S701)